code/mapping/ext_modules/mapping_common/map.py

```
@@ -108,6 +110,94 @@ def get_lane_y_axis_intersections(self, direction: str = "left") -> dict:
 
+    def entities_in_area(self, radius, only_in_front=True) -> List[Entity]:
+        """Returns the entities without the hero car
+            in a given radius around the hero car.
+            Flag only_in_front decides if entities
+            behind the car, are returend too.
+        Only checks if the first entity is_hero
+
+        Returns:
+        - Polygon: A Shapely Polygon representing the widened curve."""
+
+        if len(points) < 2:
```

Ruff (0.8.2)

Undefined name `points`

(F821)